### ISSUE TITLE
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

### ISSUE THREAD

**Issue by Jayinem** [Zer0xFF/PS4_db_rebuilder#9]
I am using the 6.7.2 fixed PS4_db_rebuilder found in another issue as I am on 6.7.2 and need to get my FPKGs back because I rebuild database. I connected to PS4 FTP, I installed python with path and run python fix_db.py 192.168.0.16 

Traceback (most recent call last):
  File "fix_db.py", line 51, in <module>
    ftp.connect(PS4_IP, 21, timeout=30)
  File "C:\Users\AppData\Local\Programs\Python\Python38-32\lib\ftplib.py", line 152, in connect
    self.sock = socket.create_connection((self.host, self.port), self.timeout,
  File "C:\Users\AppData\Local\Programs\Python\Python38-32\lib\socket.py", line 808, in create_connection
    raise err
  File "C:\Users\AppData\Local\Programs\Python\Python38-32\lib\socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

**Comment by Jayinem** [comment]
Just for the heck of it I turned my PS4 off and retried and it gave me the same error. I can't figure out why it wont connect i even opened the ip and port in my firewall to make sure.

**Comment by Jayinem** [comment]
I am pulling my hair out here I just need it to work 1 time I tried on another computer connected through ethernet and I get a different error

Traceback (most recent call last):
  File "fix_db.py", line 10, in <module>
    from sfo.sfo import SfoFile as SfoFile
  File "C:\Users\Pawn\Downloads\fix_db for 6.72\sfo\__init__.py", line 10, in <module>
    from .sfo import SfoFile
  File "C:\Users\Pawn\Downloads\fix_db for 6.72\sfo\sfo.py", line 232
    def __init__(self, sfo: SfoFile, key):
SyntaxError: invalid syntax

I can connect to my PS4 through Filezilla just fine.

**Comment by Jayinem** [comment]
Ok so now another thing, I figured out even though Python3 was in my start menu it wasn't installed on this 2nd computer so I installed it and SUCCESS! on release 0.1 that doesn't work on 6.72. I tried the fixed version that someone else posted in the other issue and it goes back to ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

It's like it's torturing me.
